### ISSUE TITLE
Value box on vertical linear gauge

### ISSUE THREAD

**Issue by OgreTransporter** [Mikhus/canvas-gauges#109]
Hi,

It would be nice to have the value box on vertical linear gauges, too.

**Comment by PetroGolovnya** [comment]
Vertical linear gauge has a value box, horisontal have not.

**Comment by OgreTransporter** [comment]
My fault! It would be nice to have a box on **horizontal** linear gauges, too.

**Comment by brblord** [comment]
I agree, would love to have the value box on the horizontal gauges as well.  A top/bottom/left/right position, padding, or offset for it would be great as well to adjust for collisions.
